backend/FwLite/LcmCrdt/Changes/SetOrderChange.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Text.Json.Serialization;
+using SIL.Harmony.Changes;
+using SIL.Harmony.Entities;
+
+namespace LcmCrdt.Changes;
+
+public class SetOrderChange<T> : EditChange<T>, ISelfNamedType<SetOrderChange<T>>
+    where T : class, IOrderable
+{
+    public static IChange Between(Guid entityId, T left, T right)
+    {
+        return new SetOrderChange<T>(entityId, (left.Order + right.Order) / 2);
+    }
+
+    public static IChange After(Guid entityId, T previous)
+    {
+        return new SetOrderChange<T>(entityId, previous.Order + 1);
+    }
+
+    public static IChange Before(Guid entityId, T preceding)
+    {
+        return new SetOrderChange<T>(entityId, preceding.Order - 1);
+    }
+
```

Comment:
I'm not actually using these, because I don't generate a SetOrderChange when a sense is created.
(I.e. if a sense is added in FieldWorks and moved to the top, it would just generate a create-sense change with the order set from the get go to position the sense at the beginning.)

So, I need the same logic outside of this change class.

So, I should probably remove these.